newt/toolchain/compiler.go

```diff
@@ -152,6 +153,36 @@ func (c *Compiler) GetLocalCompilerInfo() CompilerInfo {
 	return c.lclInfo
 }
 
+func (c *Compiler) ReplaceWildcardFlags() {
+	for _, wildcard := range c.settings.Names() {
+
+		for i, flag := range c.lclInfo.Lflags {
+			if strings.Contains(flag, "$("+wildcard+")") {
+				c.lclInfo.Lflags[i] = strings.ReplaceAll(flag, "$("+wildcard+")", c.settings.Get(wildcard))
```

Comment:
would be better to match all `$(...)` in string and then replace each with corresponding syscfg value. this also allows to emit an error if referenced syscfg does not exist.